packages/spor-theme-react/src/components/button.ts

```diff
@@ -0,0 +1,168 @@
+import type {
```

Comment:
This file is how you specify custom styles. You have some base styles, and then you have the different variants and sizes you want to support.